config/hazelcast-client.yaml

```diff
@@ -21,4 +21,4 @@ hazelcast-client:
     hazelcast.logging.type: log4j2
   network:
     cluster-members:
-      - localhost:5801
\ No newline at end of file
+      - localhost:9000
```

Comment:
Why change the port?

Reply:
> Why change the port?

refer to https://docs.hazelcast.com/imdg/4.2/clusters/advanced-network-configuration#latest-banner

Each endpoint configuration that defines a server socket must bind to a different socket address.

Reply:
> > Why change the port?
> 
> refer to https://docs.hazelcast.com/imdg/4.2/clusters/advanced-network-configuration#latest-banner
> 
> Each endpoint configuration that defines a server socket must bind to a different socket address.

Ok, Can you update the document of zeta cluster deployment?

Reply:
@Hisoka-X